.github/workflows/discord_pr_announce.yml

```diff
@@ -0,0 +1,29 @@
+name: 'New PR Notification'
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check for DISCORD_WEBHOOK"
+        id: secrets_set
+        env:
+          ENABLER_SECRET: ${{ secrets.DISCORD_WEBHOOK }}
+        run: |
+          unset SECRET_EXISTS
+          if [ -n "$ENABLER_SECRET" ]; then SECRET_EXISTS=true ; fi
+          echo "SECRETS_ENABLED=$SECRET_EXISTS" >> $GITHUB_OUTPUT
+      - name: Send Discord notification
+        uses: TiviPlus/discord-notify@v2
```

Comment:
I think something like this really should be in the `tgstation` organization for redundancy and organizational sake- you are a TGMC maintainer so I don't really think trust is a part of it but it is just valuable for head coders to be able to manage the repo and assign people to it, etc. I would advise you reach out to head coders about getting it ported in and we'll see what they say.

Reply:
Repository related actions won't be added to tgstation-operations org

It can probably be put on the tgstation org

Reply:
he said tgstation not tgstation-ops?

also yea I was assuming they'd wanna review it first or something but ill get it done

Reply:
https://github.com/tgstation/discord-notify